bin/hcl2tojson

```diff
@@ -44,8 +52,15 @@ if __name__ == '__main__':
                 processed_files.add(in_file_path)
                 processed_files.add(out_file_path)
 
-                with open(in_file_path, 'r') as in_file, open(out_file_path, 'w') as out_file:
-                    print(in_file_path)
-                    json.dump(load(in_file), out_file)
+                with open(in_file_path, 'r') as in_file:
+                    print(in_file_path, file=sys.stderr, flush=True)
+                    try:
+                        t = load(in_file)
```

Comment:
This probably could be named better. Maybe something like this below?

```suggestion
                        json_output = load(in_file)
```

Reply:
Thanks for the note, I've renamed it to a parsed_data variable as it actually contains python dict.